tests/support/rabbitmq/__init__.py

```diff
@@ -23,7 +26,9 @@
 _overview_request_path = "/api/overview"
 
 
-def current_queue_configuration(queue_name, host="localhost", port=15672) -> Optional[CurrentDestinationStatus]:
+def current_queue_configuration(
+    queue_name: str, host: str = settings.STOMP_SERVER_HOST, port: str = settings.STOMP_SERVER_INTERFACE_PORT
```

Comment:
**suggestion:** Same thing here. Since these params have a default value, the type could be `Optional`.